UI/src/styles.css

```diff
@@ -5,3 +5,21 @@
 .label {
 	float: clear;
 }
+
+
+button, .div-translation, .div-translation-body {
+	cursor: pointer;
+}
+
+button:hover, .div-translation:hover, .div-translation-body:hover {
```

Comment:
Replacing .div-translation-body with .div-translation-preview would highlight the whole translation including the votes

Reply:
done